skgstat/binning.py

```diff
@@ -49,16 +50,13 @@ def binify_even_width(X, N=10, w=None, dm=None, maxlag=None, **kwargs):
     if N is None:
         N = int(maxval / w)
 
-    # TODO if the user gives w only, it is ignored because the default value for N is 10
-
     # If N is given, calculate w from
     else:
         if w is not None:
             print('Warning! w = %d is ignored because N is already given' % w)
         w = maxval / N
 
-    # binubound = np.cumsum(np.ones(N) * w)
-    binubound = np.linspace(w, N * w, N)
+    binubound = np.cumsum(np.ones(N) * w)
```

Comment:
Why still using the bin widths over the bin edges? see my comment on the same line in the previous review

Reply:
ok, right now I am realizing that both lines do exactly the same. Therefore keep any of the two lines and remove the other one. No revision needed.